pkg/crypto/key_manager.go

```diff
@@ -0,0 +1,109 @@
+package crypto
```

Comment:
@restevens402 with this change we can delete keys.go under the masa package at `/masa-finance/masa-oracle/pkg/keys.go`

Reply:
I had removed it, but I think it might have been added back through the merge. I have removed it and checked into the branch.

Reply:
Great @restevens402 👍